output/opendap_post2.sh

```diff
@@ -225,35 +225,41 @@ for server in ${SERVERS[*]}; do
    DOWNLOADPREFIX=${properties["post.opendap.${server}.downloadprefix"]}
    CATALOGPREFIX=${properties["post.opendap.${server}.catalogprefix"]}
    OPENDAPBASEDIR=${properties["post.opendap.${server}.opendapbasedir"]}
+   OPENDAPADDROOT=${properties["post.opendap.addroot"]}
    #
    #--------------------------------------------------------------------
    #  O P E N  D A P    P A T H   F O R M A T I O N
    #--------------------------------------------------------------------
    STORMNAMEPATH=null
```

Comment:
Should this be `STORMNAMEPATH=`?

Reply:
Good question. I didn't change it so should still be functioning as it was originally. I will take a closer look.

Reply:
> Should this be `STORMNAMEPATH=`?

Do you mean should we just expose `STORMNAMEPATH`'s initial value in the configuration? I re-read this as that and it might not be a bad idea. Basically rather than have `OPENDAPADDPATH`, just have it be `STORMNAMEPATH` - or do we want to combine then into a single exposed variable that sounds more like what it is, e.g., `OPENDAPROOTPATH`? (which is what I had been playing with originally).